Gemfile

```diff
@@ -28,6 +28,8 @@ group :development, :test do
   gem 'capybara'
   gem 'byebug'
   gem 'pry'
+  gem 'brakeman', require: false
```

Comment:
it's possible that you're using platform instead of code climate classic. They are "in the process of implementing" the brakeman engine for the new code climate platform (as opposed to classic) http://docs.codeclimate.com/article/304-platform-analysis-comparison